scripts/backCompatChecker.py

```diff
@@ -55,8 +55,14 @@ def extractArgumentsFromCommand(command):
 
     return argumentList
 
+# remove azure-devops extension from index (if installed)
+subprocess.run(['az', 'extension', 'remove', '-n', 'azure-devops'], shell=True)
```

Comment:
Fix 1: This will make sure we test against the latest extension version in the index.

Reply:
will this throw if extension is not present?

Reply:
Its is subprocess so script will not fail. I Just tested it here - https://dev.azure.com/ms/azure-devops-cli-extension/_build/results?buildId=44027&view=logs&j=543512b3-d0ca-5e9c-ec0b-164b21561d9e&t=4cdb99d1-90fe-5988-0df0-0605d016b7e4&l=23